office365_view.py

```diff
@@ -105,4 +105,7 @@ def display_view(provides, all_app_runs, context):
     if provides == "list rules":
         return_page = "office365_list_rules.html"
 
+    if provides == "resolve name":
+        return_page = "office365_resolve_name.html"
+
```

Comment:
is that me or that whole method is unsafe ? What if provides value won't meet any of the clauses, return uninitialized var (aka. reaises error) 

also could just return "strign_value" instead of assigning to variable, skips few checks and saves tiniest bit of memory

Reply:
About the first issue, it is used in .json in render part of action that uses it so it will only run for those actions that are handled in those if statements, however it would be quite nice to have something that would make it more 
"leek-proof".    
   
About the second part, it is method created back in 2019 and I have only extended it.

Reply:
made a typo there but you get it

Reply:
I con do something like that: 
`return f"office365_{provides.replace(' ', '_')}.html"`.  
but I have no idea if this is faster/better than simple:   
`    [...]`
    `if provides == "get email":   `
     `   return "office365_get_email.html"  `
   ` if provides == "run query":   `
    `    return "office365_run_query.html" `
`    [...]`

Reply:
I am not gonna be stubborn on which is better but looking at it it's like writing a calculator in ifs

Reply:
As there will be more than one commit from me to this connector I will address this then

Reply:
I will also stay with return f"office365_{provides.replace(' ', '_')}.html" because than wa will always return something